gcpy/benchmark/modules/run_1yr_tt_benchmark.py

```diff
@@ -289,8 +286,8 @@ def run_benchmark(config, bmk_year_ref, bmk_year_dev):
     all_months_gchp_dev = all_months_dev
 
     bmk_mons_dev = all_months_dev[bmk_mon_inds]
-    bmk_mons_gchp_dev = all_months_gchp_dev[bmk_mon_inds]
-    bmk_sec_per_month_dev = sec_per_month_dev[bmk_mon_inds]
+    #bmk_mons_gchp_dev = all_months_gchp_dev[bmk_mon_inds]
+    #bmk_sec_per_month_dev = sec_per_month_dev[bmk_mon_inds]
```

Comment:
Can these lines be removed?

Reply:
@msulprizio: Now fixed in commit 3b380be2.